src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java

```diff
@@ -176,7 +176,7 @@ public FileResource(FileService fileService, ResourceLoaderService resourceLoade
     }
 
     /**
-     * GET /files/drag-and-drop/backgrounds/:questionId/:filename : Get the background file with the given name for the given drag and drop question
+     * GET /files/drag-and-drop/backgrounds/:postId/:filename : Get the background file with the given name for the given drag and drop question
```

Comment:
I think this update is by mistake?
```suggestion
     * GET /files/drag-and-drop/backgrounds/:questionId/:filename : Get the background file with the given name for the given drag and drop question
```

Reply:
Thanks!!